cli/cli_fileshare.go

```diff
@@ -109,7 +110,12 @@ func statusLoop(fileshareClient pb.FileshareClient, client transferStatusClient,
 // IsFileshareDaemonReachable returns error if fileshare daemon is not reachable, daemon not running
 // being the most likely cause
 func (c *cmd) IsFileshareDaemonReachable(ctx *cli.Context) error {
-	_, err := c.fileshareClient.Ping(context.Background(), &pb.Empty{})
+	resp, err := c.client.IsLoggedIn(context.Background(), &dpb.Empty{})
+	if err != nil || !resp.GetValue() {
+		return formatError(fmt.Errorf(MsgFileshareUserNotLoggedIn))
```

Comment:
[minor] isn't good the error `internal.ErrNotLoggedIn`? If not, maybe change also in `fileshareServiceErrorCodeToError` to return the same error message?

Reply:
`ServiceErrorCode_NOT_LOGGED_IN` is not returned anywhere anyway, so we might just delete this branch from `fileshareServiceErrorCodeToError`. It was probably used back when fileshare was just a part of main daemon.